src/subcommand/giraffe_main.cpp

```diff
@@ -1739,10 +1769,12 @@ int main_giraffe(int argc, char** argv) {
     // Don't try and use all the memory.
     // TODO: add memory options like autoindex?
     registry.set_target_memory_usage(IndexRegistry::get_system_memory() / 2);
-    
-    auto index_targets = map_long_reads
-                       ? VGIndexes::get_default_long_giraffe_indexes()
-                       : VGIndexes::get_default_short_giraffe_indexes();
+    auto index_targets = VGIndexes::get_default_short_giraffe_indexes();
+    if (map_long_reads) {
+        index_targets = use_path_minimizer
+                      ? VGIndexes::get_default_long_path_giraffe_indexes()
+                      : VGIndexes::get_default_long_giraffe_indexes();
+    }
```

Comment:
It would be neater if we could avoid a set-and-overwrite here. Probably we should type `index_targets` and default-construct it, then fill it in with a bunch of if/else.